src/main/java/uk/gov/ons/census/casesvc/config/MessageConsumerConfig.java

```diff
@@ -343,26 +354,4 @@ private SimpleMessageListenerContainer setupListenerContainer(
     container.setAdviceChain(retryOperationsInterceptor);
     return container;
   }
-
-  private AmqpInboundChannelAdapter makeAdapter(
```

Comment:
Just curious why you moved removed this?

Reply:
Good spot. I messed up a merge.